fossilize.cpp

```diff
@@ -3702,6 +3721,30 @@ bool StateReplayer::Impl::parse_sampler_reduction_mode(const Value &state,
 	return true;
 }
 
+bool StateReplayer::Impl::parse_input_attachment_aspect(const Value &state,
+						        VkRenderPassInputAttachmentAspectCreateInfo **out_info)
+{
+	auto *info = allocator.allocate_cleared<VkRenderPassInputAttachmentAspectCreateInfo>();
+	*out_info = info;
+
+	info->aspectReferenceCount = state["aspectReferenceCount"].GetUint();
```

Comment:
This is implied by state["aspectReferences"].Size().

Reply:
Should I remove aspectReferenceCount and rely on the array size?

Reply:
Yes.

Reply:
Well, it must be greater than 0 according to the spec.

Reply:
Ok, probably ignore the test about 0 count then.

Reply:
Should be addressed now hopefully.